src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoParticipantData.java

```diff
@@ -0,0 +1,93 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBRangeKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.sagebionetworks.bridge.models.ParticipantData;
+
+import java.util.Objects;
+
+/**
+ *
+ */
+@DynamoDBTable(tableName = "ParticipantData")
+public class DynamoParticipantData implements ParticipantData {
+
+    private String userId;
+    private String configId;
+    private String data;
+    private Long version;
+
+    //TODO: figure out which methods require a @JsonIgnore annotation
+    //TODO: figure out which methods require a @DynamoDBIgnore annotation
```

Comment:
Only the healthCode hash key column needs to be suppressed. I don't think there's any reason to ever expose it, so @JsonIgnore would work.